doc/source/security.rst

```diff
@@ -12,7 +12,15 @@ This makes things quick and easy for new users to get up and running, but may po
 
 Many organisations have policies which do not allow users to assign public IP addresses or open ports. Our best practices
 advice is to use Dask Cloudprovider from within a cloud platform, either from a VM or a managed environment. Then disable public
-networking.
+networking. For example: 
```

Comment:
We presumed this is what "disable public networking" meant. I felt a concrete example helped walk this out a bit.